docs/examples/how-tos.md

```diff
@@ -0,0 +1,396 @@
+
+Enter the `CKEDITOR_VERSION` command to check the currently used CKEditor 5 version.
+
+{@img assets/img/version.png 468 CKEditor 5 version displayed in the developer console.}
+
+## Editor's instances
+
+### How to get the editor instance in the plugin?
+
+In a simple plugin, you can get the editor's instance using a plugin functions' attribute
+
+```js
+function MyPlugin(editor) {
+	// Interact with the API.
+	// ...
+}
+
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	// If you're using builds, this is going to be extraPlugins property.
+	plugins: [
+		myPlugin
+		// Other plugins.
+		// ...
+	]
+} );
+```
+
+### How to get the editor instance object from the DOM element?
+
+If you have a reference to the editor editable's DOM element (the one with the `.ck-editor__editable` class and the `contenteditable` attribute), you can access the editor instance this editable element belongs to using the `ckeditorInstance` property:
+
+```html
+<!-- The editable element in the editor's DOM structure. -->
+<div class="... ck-editor__editable ..." contenteditable="true">
+	<!-- Editable content. -->
+</div>
+```
+
+```js
+// A reference to the editor editable element in the DOM.
+const domEditableElement = document.querySelector( '.ck-editor__editable' );
+
+// Get the editor instance from the editable element.
+const editorInstance = domEditableElement.ckeditorInstance;
+
+// Use the editor instance API.
+editorInstance.setData( '<p>Hello world!<p>' );
+```
+
+### How to list all instances of the editor?
+
+By default, CKEditor 5 has no global registry of editor instances. But if necessary, such a feature can be easily implemented, as explained in this [Stack Overflow answer](https://stackoverflow.com/a/48682501/1485219).
+
+## Editor's API
+
+### How to insert some content into the editor?
+
+Because CKEditor 5 uses a custom {@link framework/architecture/editing-engine data model}, whenever you want to insert anything, you should modify the model first, which is then converted back to the view where the users input their content (called "editable"). In CKEditor 5, HTML is just one of many possible output formats. You can learn more about the ways of changing the model in the {@link framework/architecture/editing-engine#changing-the-model dedicated guide}.
+
+For example, to insert a new link at the current position, use the following snippet:
+
+```js
+editor.model.change( writer => {
+	const insertPosition = editor.model.document.selection.getFirstPosition();
+
+	writer.insertText(
+		'CKEditor 5 rocks!',
+		{ linkHref: 'https://ckeditor.com/' },
+		insertPosition
+	);
+} );
+```
+
+And to insert some plain text, you can use a slightly shorter one:
+
+```js
+editor.model.change( writer => {
+	writer.insertText(
+		'Plain text',
+		editor.model.document.selection.getFirstPosition()
+	);
+} );
```

Comment:
This code (in both snippets) is not verifying the model schema and should not be used in the simplest snippets. We should promote `model.insertContent()` or for widgets `model.insertObject()`.

Reply:
Ok, to fix.